other_topics/python_decorators/1_functions_in_python.py

```diff
@@ -0,0 +1,114 @@
+"""
+1. functions in Python.
+What you need to know about functions in Python for decorators.
+"""
+
+
+"""
+In python, functions are objects, therefore they can be used in many ways:
```

Comment:
suggest to add, "functions are first class citizens in python" and this link [https://en.wikipedia.org/wiki/First-class_function](https://en.wikipedia.org/wiki/First-class_function).